app/models/samples/json/model_with_humidity_setpoints.json

```diff
@@ -413,6 +413,88 @@
                 }
             ],
             "schedules": [
+                {
+                    "type": "ScheduleRulesetAbridged",
+                    "name": "Hospital INFIL_SCH_PNNL",
+                    "default_day_schedule": {
```

Comment:
@chriswmackey , this is missing the field day_schedules and does not have referencing for schedule day in default day schedule, summer and winter design day schedule